src/hats/io/parquet_metadata.py

```diff
@@ -302,13 +324,18 @@ def per_pixel_statistics(
         exclude_columns (List[str]): additional columns to exclude from the statistics.
         include_columns (List[str]): if specified, only return statistics for the column
             names provided. Defaults to None, and returns all non-hats columns.
+        only_numeric_columns (bool): only include columns that are numeric (integer or
+            floating point) in the statistics. If True, the entire frame should be numeric.
         include_pixels (list[HealpixPixel]): if specified, only return statistics
             for the pixels indicated. Defaults to none, and returns all pixels.
         include_stats (List[str]): if specified, only return the kinds of values from list
             (min_value, max_value, null_count, row_count). Defaults to None, and returns all values.
         multi_index (bool): should the returned frame be created with a multi-index, first on
             pixel, then on column name? Default is False, and instead indexes on pixel, with
             separate columns per-data-column and stat value combination.
+        per_row_group (bool): should the returned data be even more fine-grained and provide
+            per row group (within each pixel) level statistics? Default is currently False,
+            but is very likely to change.
```

Comment:
What makes it likely to change? Just that row group statistics tend to have more uses?